tests/integration/master/test_clear_funcs.py

```diff
@@ -15,85 +16,78 @@
 from tests.support.mixins import AdaptedConfigurationTestCaseMixin
 from tests.support.runtests import RUNTIME_VARS
 
+log = logging.getLogger(__name__)
+
+
+class ConfigMixin:
+    @classmethod
+    def setUpClass(cls):
+        cls.master_config = AdaptedConfigurationTestCaseMixin.get_config("master")
+        cls.minion_config = AdaptedConfigurationTestCaseMixin.get_temp_config(
+            "minion",
+            id="root",
+            transport="zeromq",
```

Comment:
The transport needs to change if we're running the tcp transport tests.

https://github.com/saltstack/salt/pull/57188/files#diff-e0fc5514059fe3cbc849239025bf09f3R198

Reply:
Fixed. Thanks!